src/api/app/models/bs_request.rb

```diff
@@ -703,9 +703,9 @@ def change_review_state(new_review_state, opts = {})
       history.create(p) if history
 
       # we want to check right now if pre-approved requests can be processed
-      if go_new_state == :new and self.accept_at
-        Delayed::Job.enqueue AcceptRequestsJob.new
-      end
+      # if go_new_state == :new and self.accept_at
+      #   Delayed::Job.enqueue AcceptRequestsJob.new
```

Comment:
I gues that it only needs to use ::AcceptRequestJob to not be in the namespace of BsRequest because the model is already defined as a worker in open-build-service/src/api/lib/workers/accept_requests.rb

And adding a require to the file first:
```ruby
require File.join(Rails.root, 'lib/workers/accept_requests.rb')
```